src/hostname/hostnamed.c

```diff
@@ -379,6 +379,10 @@ static int get_firmware_vendor(Context *c, char **ret) {
         return get_dmi_sysattr(c, "bios_vendor", ret);
 }
 
+static int get_chassis_asset_tag(Context *c, char **ret) {
```

Comment:
Not a big matter, but please move this slightly above around functions which uses get_dmi_properties() or get_dmi_property().